server/tests/backends/fixtures/aggregate/last_pypika.json

```diff
@@ -0,0 +1,58 @@
+{
+  "exclude": [
+    "mongo",
+    "mysql",
+    "pandas",
+    "postgres",
+    "snowflake"
+  ],
+  "step": {
+    "pipeline": [
+      {
+        "name": "aggregate",
+        "on": [
+          "beer_kind"
+        ],
+        "aggregations": [
+          {
+            "newcolumns": [
+              "last_name"
+            ],
+            "aggfunction": "last",
+            "columns": [
+              "name"
+            ]
+          }
+        ],
+        "keepOriginalGranularity": false
+      }
+    ]
+  },
+  "expected": {
+    "schema": {
+      "fields": [
+        {
+          "name": "beer_kind",
+          "type": "any"
+        },
+        {
+          "name": "last_name",
+          "type": "string"
+        }
+      ],
+      "pandas_version": "1.4.0"
+    },
+    "data": [
+      {"beer_kind": "Belge blonde forte & Golden Ale", "last_name": "Pauwel Kwak"},
+      {"beer_kind": "Best-sellers", "last_name": "Ardwen Blonde"},
+      {"beer_kind": "Blanche & Weizen", "last_name": "Weihenstephan Hefe Weizen Alcoholarm"},
+      {"beer_kind": "Blonde", "last_name": "Cuvée des Trolls"},
+      {"beer_kind": "Blonde", "last_name": "Cuvée des Trolls"},
+      {"beer_kind": "Blonde", "last_name": "Cuvée des Trolls"},
```

Comment:
Shouldn't we only have one "Blonde" here since we're grouping on `beer_kind` and `keepOriginalGranularity` is false ?